build/test/get-test-paths.js

```diff
@@ -0,0 +1,35 @@
+const globby = require('globby');
+
+/**
+ * Get list of paths/ files to test
+ */
+const getTestPaths = async () => {
+	const host = `localhost`;
```

Comment:
Why is this a template string? Also, why is these variables necessary?

Reply:
Hopefully this provides some insight - https://github.com/dequelabs/html-team-proposals/pull/6/files#diff-da0741cfbe1cedb0339d04635fd2fc01R73

Reply:
Nope. Still don't understand. I think this could be removed.